app/routes.ts

```diff
@@ -27,6 +27,7 @@ export default [
       route("logout", "routes/auth/logout.tsx"),
     ]),
     ...prefix("u", [
+      route("access-token", "routes/user/access-token.tsx"),
```

Comment:
I didn't add a link to this anywhere yet. Probably somewhere on the profile page makes sense?  But felt like a lot there already